### PR TITLE
Parallellize non-system tests

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -79,9 +79,32 @@ jobs:
...
+      matrix: ${{ steps.test-files.outputs.matrix }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - id: test-files
+        env:
+          FILES_PER_BATCH: 60
+        run: |
+          tests=$(find test -name *_test.rb -not -path system | xargs -n ${{ env.FILES_PER_BATCH }} | xargs -I {} echo '"{}"' | tr '\n' ',')
```
Comment (iHiD) [inline .github/workflows/tests.yml]: From looking at the logs, system isn't getting excluded properly, which is why this is slow.

Comment (iHiD) [reply .github/workflows/tests.yml]: (e.g. https://github.com/exercism/website/runs/4395693256?check_suite_focus=true)

Comment (ErikSchierboom) [reply .github/workflows/tests.yml]: Ah, feck. It worked locally. I'll figure it out

Comment (ErikSchierboom) [reply .github/workflows/tests.yml]: Should be fixed now

**Pull request merged**